Examples/OneSignalDemo/app/src/main/java/com/onesignal/sdktest/constant/Key.java

```diff
@@ -0,0 +1,5 @@
+package com.onesignal.sdktest.constant;
+
+public class Key {
```

Comment:
Class has no internals, this used anywhere?

Reply:
Got it